Api.dockerfile

```diff
@@ -27,7 +27,7 @@ RUN dotnet publish src/Api/ \
 # ** Run
 
 # Use `runtime-deps` instead of `runtime` because we have a self-contained assembly
-FROM --platform=$TARGETPLATFORM mcr.microsoft.com/dotnet/runtime-deps:8.0-alpine AS run
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-bookworm-slim AS run
```

Comment:
Note that this results in a significantly bigger image. We went with `runtime-deps` specifically to reduce test startup time.